lib/cisco_node_utils/cmd_ref/interface_ospf.yaml

```diff
@@ -54,6 +64,12 @@ message_digest_password:
   default_value: ~
   get_value: '/^\s*ip ospf message-digest-key \d+ \S+ \d+ (\S+)/'
 
+network_type_p2p:
```

Comment:
Why `network_type_p2p` instead of just `network`?  There are four different network types. I'd prefer to see this as a string, a la:

```
network_type:
  get_value: '/^ip ospf network (\S+)$/'
  set_value: "%s ip ospf network %s"
  default_value: 'broadcast'
```